protocol/src/private_id/company.rs

```diff
@@ -315,4 +330,33 @@ impl CompanyPrivateIdProtocol for CompanyPrivateId {
             })
             .map_err(|_| ProtocolError::ErrorIO("Unable to write company view to file".to_string()))
     }
+
+    fn stringify_id_map(&self, use_row_numbers: bool) -> String {
```

Comment:
It makes more sense for it to remain a `protocol` method, rather than be imported from `common`, but I did just now move the bulk of the function into files.rs to reduce the amount duplicated code.  Do note for example, that your existing `save_id_map` and `print_id_map` are duplicates themselves, which I think is fine.